python/ambassador/diagnostics/diagnostics.py

```diff
@@ -118,8 +118,7 @@ def __init__(self, diag: "Diagnostics", estat: EnvoyStats, request) -> None:
 
         # Go ahead and grab Envoy cluster stats for all possible clusters.
         # XXX This might be a bit silly.
-        self.cluster_names = [cluster.envoy_name for cluster in self.diag.clusters.values()]
```

Comment:
Since we are assigning `self.cluster_names` did you verify that this wasn't being referenced anywhere else? I just want to make sure nothing was referencing it since it is mutating the `DiagResults`

Reply:
I've double-checked the `cluster_names`, and it seems that was created just like an aux variable, there's not any implementation elsewhere.